src/__screenshot_tests__/image-screenshot-test.tsx

```diff
@@ -1,9 +1,59 @@
 import {openStoryPage, screen} from '../test-utils';
 
-test('Image', async () => {
+const ASPECT_RATIO = ['1:1', '16:9', '7:10', '4:3', '0'];
+
+test('Image - width and height', async () => {
     await openStoryPage({id: 'components-primitives-image--default'});
 
-    const story = await screen.findByTestId('image-story');
+    const story = await screen.findByTestId('image');
+
+    expect(await story.screenshot()).toMatchImageSnapshot();
+});
+
+test.each(ASPECT_RATIO)('Image - aspect ratio %s', async (aspectRatio) => {
+    await openStoryPage({
+        id: 'components-primitives-image--default',
+        args: {
+            type: 'width and aspect ratio',
+            aspectRatio: aspectRatio.replace(':', ' '),
+        },
+    });
+
+    const story = await screen.findByTestId('image');
+
+    expect(await story.screenshot()).toMatchImageSnapshot();
+});
+
+test.each(ASPECT_RATIO)('Image - full width with aspect ratio %s', async (aspectRatio) => {
+    await openStoryPage({
+        id: 'components-primitives-image--default',
+        args: {type: 'full width', aspectRatio: aspectRatio.replace(':', ' ')},
+    });
+
+    const story = await screen.findByTestId('image');
+
+    expect(await story.screenshot()).toMatchImageSnapshot();
+});
+
+test('Image - no border radius', async () => {
+    await openStoryPage({id: 'components-primitives-image--default', args: {borderRadius: false}});
+
+    const story = await screen.findByTestId('image');
+
+    expect(await story.screenshot()).toMatchImageSnapshot();
+});
+
+test.each(ASPECT_RATIO.slice(0, 4))('Image - error fallback with aspect ratio %s', async (aspectRatio) => {
```

Comment:
just curious, why was the last ASPECT_RATIO element skipped?

Reply:
When aspectRatio is 0, it means "preserve the original aspect ratio of the image". The error fallback is shown when the image failed loading. In that scenario, there is no way to actually know the original aspect ratio, so the height of the fallback ends up being equal to 0.

I don't think we can actually avoid this, and I don't see a reason to include it in the screenshot tests